types/foundry-vtt-api/types/apps/hud/hud.d.ts

```diff
@@ -0,0 +1,29 @@
+// @TODO:
+
+/**
+ * An abstract base class for displaying a heads-up-display interface bound to a Placeable Object on the canvas
+ */
+declare class BasePlaceableHUD extends Application {
+    /**
+     * Reference a PlaceableObject this HUD is currently bound to
+     */
+    object: PlaceableObject;
+
+    /**
+     * BasePlaceableHUD display states
+     */
+    DISPLAY_STATES: object;
+
+    constructor(...args: any[]);
+
+    /**
+     * Bind the HUD to a new PlaceableObject and display it
+     * @param object	A PlaceableObject instance to which the HUD should be bound
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/hud/hud.d.ts;
  Line: 21;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.